bin/cleanup_manual_snapshots

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+echo "Logging into RDS"
+
+regex_manual='apply-manual-snapshot'
+
+regex_automatic='rds:cloud-platform'
+
+unset AWS_PROFILE
+
+read AWS_ACCESS_KEY_ID name pass user addr endp AWS_SECRET_ACCESS_KEY url \
+      <<<$(kubectl -n laa-apply-for-legalaid-staging get secret apply-for-legal-aid-rds-instance-output -o json | jq -r '.data[] | @base64d')
+
+export AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
+
+# uncomment the line below to print a list of the current backups to the console
```

Comment:
Nit picky, but the line below isn't commented out, so I'm not sure if this comment really adds anything

Reply:
now commented out